src/openrct2-ui/input/KeyboardShortcut.cpp

```diff
@@ -767,6 +768,38 @@ static void shortcut_advance_to_next_tick()
     gDoSingleUpdate = true;
 }
 
+static void shortcut_open_scenery_picker()
+{
+    if (gScreenFlags & SCREEN_FLAGS_TITLE_DEMO)
```

Comment:
```suggestion
    if ((gScreenFlags & (SCREEN_FLAGS_TITLE_DEMO | SCREEN_FLAGS_TRACK_DESIGNER | SCREEN_FLAGS_TRACK_MANAGER) || (gScreenFlags & SCREEN_FLAGS_SCENARIO_EDITOR && gS6Info.editor_step != EDITOR_STEP_LANDSCAPE_EDITOR))
```

That get rid if the nested if. Make sure you use clang-format to autoformat.